specification/maps/data-plane/DataRegistry/stable/2022-12-01/dataregistry.json

```diff
@@ -0,0 +1,716 @@
+            },
+            "x-ms-error-response": true
+          }
+        }
+      }
+    },
+    "/mapData/{udid}/metadata": {
+      "get": {
+        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nThe endpoint of DataRegistry APIs is bound to the region of your maps account. Global maps account is not be able to use this service.\n\nThis API allows the caller to fetch the metadata of a given udid registered previously using the [DataRegistry Register API](https://docs.microsoft.com/en-us/rest/api/maps/data-registry/register). If Azure Maps cannot access the underlying data or integrity check is not passed, `status` will be `failed` and return the error details.\n\n## Submit Get Metadata Request\n\nTo fetch the metadata of a given udid you will issue a `GET` request with no additional parameters.",
+        "operationId": "DataRegistries_GetMetadata",
+        "x-ms-examples": {
+          "Fetch the metadata of a given valid udid": {
+            "$ref": "./examples/DataRegistries_GetMetadataValid.json"
+          },
+          "Fetch the metadata of a given invalid udid": {
+            "$ref": "./examples/DataRegistries_GetMetadataInvalid.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/UniqueDataId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Get metadata request completed successfully. The response body is the metadata of a given data registry.",
+            "schema": {
+              "$ref": "#/definitions/MetadataResult"
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        }
+      }
+    },
+    "/mapData/operations/{operationId}": {
+      "get": {
+        "description": "The endpoint of DataRegistry APIs is bound to the region of your maps account. Global maps account is not be able to use this service.\n\nThis path will be obtained from a call to [DataRegistry Register API](https://docs.microsoft.com/en-us/rest/api/maps/data-registry/register), [DataRegistry Update API](https://docs.microsoft.com/en-us/rest/api/maps/data-registry/update), and [DataRegistry Patch API](https://docs.microsoft.com/en-us/rest/api/maps/data-registry/patch). While in progress, a HTTP `200 OK` response will be returned with no extra headers -  followed by a HTTP `200 OK` with `Resource-Location` header once completed.\n\n## Get Metadata Response\n\nThe Data Metadata API returns the complete list of all data in `json` format.",
```

Comment:
>This path will be obtained from a call to [DataRegistry Register API](https://docs.microsoft.com/en-us/rest/api/maps/data-registry/register), [DataRegistry Update API](https://docs.microsoft.com/en-us/rest/api/maps/data-registry/update), and [DataRegistry Patch API](https://docs.microsoft.com/en-us/rest/api/maps/data-registry/patch)

we don't have UPDATE anymore.
Do we return operation id in Patch? I assume we only return it in LRO and the only LRO we have now is Register API.

Reply:
Fixed